elks/tools/objtools/ewcc

```diff
@@ -33,6 +33,7 @@ WATCINCLUDE=$WATCOM/h
 # -march=i86                # 8086 codegen
 # -std=c99                  # -Wc,-za99
 # -Wc,-zev                  # enable void arithmetic
+# -Wc,-zls                  # remove automatically inserted symbols
```

Comment:
Is there a list of symbols OWC might automatically insert? These could be useful in various circumstances for ELKS libc, just like they are in OWC libc.

Reply:
No, but below is list from C compiler source code
"__DLLstartw_"
"__DLLstart_"
"_wstartw_"
"_wstart_"
"_cstartw_"
"_cstart_"
"_fltused_80bit_"
"_fltused_"
"_big_code_"
"_small_code_"
"__init_87_emulator"
"__init_387_emulator"
"__old_8087"
"__8087"
"_fltused_"
"__wargc"
"__argc"
"__wargc"
"__argc"
"_wargc"
"_argc"
"__init_default_win"
"__WATCOM_Prelude"
"__p5_profile"
"__new_p5_profile"

Take into account that it is full list but they are genearted in dependncy of compiled code or target platform, some are specific to architecture etc. Best is look into compiler code because the is condition when it is used.
Anyway now it is suppresed by -zls option that none of this symbols is included in generated code.

Reply:
Thanks for the list. Approved. In the future, it may be good to have OW insert `fltused` or similar, but we'll worry about that later.